src/Chainweb/BlockHeaderDB/PruneForks.hs

```diff
@@ -1,7 +1,16 @@
-{-# LANGUAGE FlexibleContexts #-}
 {-# LANGUAGE BangPatterns #-}
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE DeriveGeneric #-}
+{-# LANGUAGE DerivingStrategies #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE GADTs #-}
```

Comment:
I don't see a GADT declared, is this for usage of a GADT, or RankNTypes, or?

Reply:
I think, it's not needed any more and can be removed. But not 100% sure.